internal/operands/template-validator/resources.go

```diff
@@ -70,12 +62,13 @@ func newServiceAccount(namespace string) *core.ServiceAccount {
 func newClusterRoleBinding(namespace string) *rbac.ClusterRoleBinding {
 	return &rbac.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   ClusterRoleBindingName(namespace),
-			Labels: commonLabels,
+			Name:      ClusterRoleBindingName,
+			Namespace: namespace,
```

Comment:
The same here. The namespace needs to be empty.

Reply:
Done